docs/UserGuide.md

```diff
@@ -141,15 +142,15 @@ An example of a command in TAB:
 #### Command format notation
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `add n/NAME`, `NAME` is a parameter that can be substituted as `add n/John Doe`.
```

Comment:
emm "`NAME` is a parameter that can be substituted..." sounds like substituting just the NAME but the eg included the prefix as well
```suggestion
  e.g. in `add n/NAME`, `NAME` is a parameter that can be substituted with `John Doe`.
```